include/tvm/relay/transform.h

```diff
@@ -580,6 +580,14 @@ TVM_DLL Pass AnnotateUsedMemory();
  */
 TVM_DLL Pass CapturePostDfsIndexInSpans();
 
+/*!
+ * \brief Remove reshapes after lowering the graph.
+ *
+ *
+ * \return The pass.
+ */
+TVM_DLL Pass RemoveReshapes();
```

Comment:
Lets rename this to remove StandaloneReshapes

Reply:
ACK